### PR TITLE
Smart Swap documentation improvements 

### PR DESCRIPTION
**Pull request by ericHgorski**
Improved copy, organization, and code examples

### CODE REVIEW

File: docs/advanced-swapping/smart-swap-options.mdx [hunk 1]
```diff
@@ -90,30 +91,64 @@ const route = await skipClient.route({
...
+| Base          | 8453     |
+| BNB Chain     | 56       |
+| Avalanche     | 43114    |
+| Blast         | 81457    |
+| Celo          | 42220    |
 
-Just set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on version 5.1.0 or above (we advise you migrate over to `@skip-go/client` as soon as possible).
+### Usage
+
+Set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on v5.1.0+ (we strongly reccomend migrating to `@skip-go/client` as soon as possible).
```
Comment (NotJeremyLiu) [inline docs/advanced-swapping/smart-swap-options.mdx:148]:
```suggestion
Set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on v5.1.0+ (we strongly recommend migrating to `@skip-go/client` as soon as possible).
```

File: docs/advanced-swapping/smart-swap-options.mdx [hunk 2]
```diff
@@ -90,30 +91,64 @@ const route = await skipClient.route({
...
+export type SwapRoute = {
+  swapAmountIn: string;
+  denomIn: string;
+  swapOperations: SwapOperation[];
+};
+```
+
+## Feature: EVM Swaps
+
+Smart Swap supports bidrectional EVM swaps: go from any asset on an EVM chain to any asset on a Cosmos chain and back again. With EVM swaps, users can onboard to your IBC connected chain in 1 transaction from a broad range of EVM assets, including the memecoins retail loves to hold!
```
Comment (NotJeremyLiu) [inline docs/advanced-swapping/smart-swap-options.mdx:130]: fire bar

File: docs/advanced-swapping/smart-swap-options.mdx [hunk 3]
```diff
@@ -90,30 +91,64 @@ const route = await skipClient.route({
...
+| Blast         | 81457    |
+| Celo          | 42220    |
 
-Just set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on version 5.1.0 or above (we advise you migrate over to `@skip-go/client` as soon as possible).
+### Usage
+
+Set the `evmSwaps` flag to true in the `smartSwapOptions` object. If using the deprecated `@skip-router` library, you must be on v5.1.0+ (we strongly reccomend migrating to `@skip-go/client` as soon as possible).
 
 <CodeGroup>
-```Typescript TypeScript
```
Comment (NotJeremyLiu) [inline docs/advanced-swapping/smart-swap-options.mdx:116]: This seems to cause rendering issues on github preview, idk if the same on mintlify

File: docs/advanced-swapping/smart-swap-options.mdx [hunk 4]
```diff
@@ -5,56 +5,45 @@ description: 'This page introduces the Smart Swap functionality provided by the
...
 
-1. `@skip-go/client` Package: `smartSwapOptions` object passed in`SkipClient.route` options `SkipClient.msgsDirect` options
-2. REST API: `smart_swap_options` parameter passed in `/v2/fungible/route` and `/v2/fungible/msgs_direct`
+The rest of this document will show you how to use Smart Swap with the `@skip-go/client` library. The only changes you'll notice between this context and the REST API are naming conventions.
 
-Read on for more info about the individual features and how to use each of them.
+# Smart Swap Features
 
-(For the rest of this doc, examples will show how to use Smart Swap with the `@skip-go/client` library by configuring it on `SkipClient.route`. The only changes you'll notice between this context and the REST API is naming conventions)
+Set your Smart Swap settings in your `skipClient` function call our REST API request body.
```
Comment (NotJeremyLiu) [inline docs/advanced-swapping/smart-swap-options.mdx:27]: I think there's some typo here? the sentence doesn't make sense to me

**Review by NotJeremyLiu — APPROVED** [review]
left some wording comments/nits, approving now, feel free to merge once feedback is addressed / ping me to re-review if you'd like

File: docs/advanced-swapping/smart-swap-options.mdx (hunk 1 above)
Comment (ericHgorski) [reply docs/advanced-swapping/smart-swap-options.mdx:148]: installing a spell checker rn :---)

File: docs/advanced-swapping/smart-swap-options.mdx (hunk 4 above)
Comment (ericHgorski) [reply docs/advanced-swapping/smart-swap-options.mdx:27]: good catch, should just be removed.

**Pull request merged**